.github/workflows/refresh_knapsack_manifest.yml

```diff
@@ -0,0 +1,58 @@
+name: "Refresh Knapsack manifest"
+
+on:
+  schedule:
+    - cron: '0 8 * * 1' # Monday at 8am
+  workflow_dispatch: 
+  push: # Temp for testing
```

Comment:
All good @ethax-ross 👍 Just a reminder to remove this line before merging.. tks 😄